build/lib/pgBackRestBuild/Config/Data.pm

```diff
@@ -2843,6 +2861,18 @@ foreach my $strKey (sort(keys(%hConfigDefine)))
         {
             $hConfigDefine{$strKey}{$strOptionDef} = $hConfigDefineOverride->{$strOptionDef};
         }
+
+        # Update option variable with new hash reference
+        $rhOption = $hConfigDefine{$strKey}
+    }
+
+    # If the option group is defined then copy configuration from the group to the option
+    if (defined($rhOption->{&CFGDEF_GROUP}))
```

Comment:
I'm assuming since that we copy the group stuff into the global structure when cfgDefine() is called, that it was not necessary to change the code dealing with the group indexed items (e.g. lines 44+) in DocExecute.pm - correct?

Reply:
Correct.